src/react-intl-messages.ts

```diff
@@ -815,7 +823,9 @@ export default {
         [FM.BUTTON_IMPORT]: 'Import',
         [FM.BUTTON_INFO]: 'Info',
         [FM.BUTTON_LOAD]: 'Load',
+        [FM.BUTTON_MORE_LOGS]: 'More Logs',
```

Comment:
I think the button should have the verb `Load` to explain what it does. Clicking it will load more.  More logs is still ambigous